src/components/auth/login.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button, Typography } from '@material-ui/core';
+import MuiAlert, { AlertProps } from '@material-ui/lab/Alert';
+import React from 'react';
+import { Redirect } from 'react-router';
+import appConfig from '../../app-config';
+import authStyles from './auth-style';
+
+export function Alert(props: AlertProps): React.ReactElement {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+export interface LoginProps {}
+
+export default function Login(_props: LoginProps): React.ReactElement {
+  const authenticator = appConfig.authenticator;
+  const classes = authStyles();
+  const [loginResponse, setLoginResponse] = React.useState(false);
+
+  async function handleRmfLogin(_event: React.MouseEvent): Promise<void> {
+    const redirectUri = new URL(window.location.href);
+    redirectUri.searchParams.append('response', '1');
```

Comment:
Could you comment a bit about the purpose of this?

It's for getting and setting the response from the authenticator in a place right?

Related
```
function getRedirectUri(): URL {
  const redirectUri = new URL(window.location.href);
  redirectUri.pathname = LOGIN_ROUTE;
  redirectUri.searchParams.append('response', '1');
  return redirectUri;
}
```

Reply:
thanks for the catch! This is leftovers from some refactors, it should now use the url from appConfig.